lib/command/init.rb

```diff
@@ -70,7 +79,11 @@ def init_aozoraepub3(force = false)
         puts "<bold><red>#{"!!!WARNING!!!".center(70)}</red></bold>".termcolor
         puts "AozoraEpub3の構成ファイルを書き換えます。narouコマンド用に別途新規インストールしておくことをオススメします"
       end
-      aozora_path = ask_aozoraepub3_path
+      if @aozora_dirname then
+        aozora_path = @aozora_dirname
```

Comment:
__[rubocop]__
* Style/IndentationWidth: Use 2 (not -12) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)

<!-- Comment by [SideCI](https://sideci.com) -->